themes/mongodb/src/widgets/suggestion/suggestion.js

```diff
@@ -0,0 +1,81 @@
+import PropTypes from 'prop-types';
+import {Stitch} from 'mongodb-stitch-browser-sdk';
+import SuggestionCardList from './SuggestionCardList';
+import preact from 'preact';
+
+const appID = 'ref_data-bnbxq';
+const client = Stitch.initializeDefaultAppClient(appID);
+
+function getPageName() {
+    const bodyElements = document.getElementsByClassName('body');
+    if (!bodyElements.length) { return null; }
+
+    const pagename = bodyElements[0].getAttribute('data-pagename');
+
+    return pagename;
+}
+
+class Suggestion extends preact.Component {
+    constructor(props) {
+        super(props);
+
+        this.handleCloseDrawer = this.handleCloseDrawer.bind(this);
+        this.pageName = getPageName();
+
+        this.state = {
+            'isLoaded': false,
+            'suggestions': []
+        };
+    }
+
+    componentDidMount() {
+        client.callFunction('fetchSuggestions', [this.pageName]).then(
+            (result) => {
+                this.setState({
+                    'isLoaded': true,
+                    'suggestions': result
+                });
+            },
+            (error) => {
+                console.log('error');
+                this.setState({
+                    'isLoaded': true
+                });
+            }
+        );
+    }
+
+    handleCloseDrawer(source) {
+        this.props.handleCloseDrawer();
+        ga('send', { // eslint-disable-line no-undef
```

Comment:
Last I checked, we didn't submit anything to GA directly; we used `utils.reportAnalytics()` to send events to Segment, which would then syndicate to GA. If you've heard otherwise from kirby, then this is fine